#! /bin/sh /usr/share/dpatch/dpatch-run
## debian_jail.dpatch by  <debian@onerussian.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Default debian jails

@DPATCH@

diff -Naur -x '*~' fail2ban-0.7.1/config/jail.conf fail2ban-0.7.1.modified/config/jail.conf
--- fail2ban-0.7.1/config/jail.conf	2006-08-20 17:33:11.000000000 -0400
+++ fail2ban-0.7.1.modified/config/jail.conf	2006-09-05 10:56:44.000000000 -0400
@@ -5,11 +5,20 @@
 # $Revision: 281 $
 #
 
-[dummy]
+[SSH]
 
-enabled = false
+enabled = true
 filter = sshd
-action = hostsdeny[file=/tmp/hosts.deny]
-         mail[name=SSH, dest=cyril.jaquier@bluewin.ch]
-maxretry = 2
+action = iptables
+         mail[name=SSH, dest=root@localhost]
+maxretry = 5
+bantime = 10
+
+[Apache]
+
+enabled = false
+filter = apache-auth
+action = iptables
+         mail[name=Apache, dest=root@localhost]
+maxretry = 5
 bantime = 10
